internal/expr/typeinfo.go

```diff
@@ -115,6 +115,18 @@ func (mi *mapInfo) getAllMembers() ([]typeMember, error) {
 	return nil, fmt.Errorf(`map type %q cannot be used with asterisk`, mi.mapType.Name())
 }
 
+type sqlContext int
```

Comment:
For now I think we could just have a boolean here, that indicates if it is an insert statement. If we need more cases in the future we could add something like this.

Reply:
Changed